concent_api/core/tests/test_payments_sci_backend.py

```diff
@@ -254,3 +267,31 @@ def test_that_sci_backend_cover_additional_verification_cost_should_return_trans
             value=self.transaction_value,
             subtask_id=sci_backend._hexencode_uuid(self.task_to_compute.subtask_id),
         )
+
+    def test_that_if_there_is_no_previous_transactions_validation_should_pass(self):  # pylint: disable=no-self-use
+        last_payment_closure_time_timestamp = get_current_utc_timestamp() - 3600 * 24 * 10  # 10 days ago
+        with mock.patch('core.payments.validation.get_list_of_payments', return_value=[]):
+            validate_that_last_closure_time_is_older_than_oldest_payment(
+                'any_key', 'any_key', last_payment_closure_time_timestamp
+            )
+
+    def test_that_if_there_is_no_payment_younger_then_oldest_payment_timestamp_in_subtask_results_accepted_validation_should_pass(self):  # pylint: disable=no-self-use
+        last_payment_closure_time_timestamp = get_current_utc_timestamp() - 3600 * 24 * 10  # 10 days ago
+        with mock.patch(
+            'core.payments.validation.get_list_of_payments',
+            return_value=_prepare_forced_payment_event_list_without_younger_payments(
+                last_payment_closure_time_timestamp)):
+            validate_that_last_closure_time_is_older_than_oldest_payment(
+                'any_key', 'any_key', last_payment_closure_time_timestamp
+            )
+
+    def test_that_if_there_is_payment_younger_then_oldest_payment_timestamp_in_subtask_results_accepted_validation_should_raise_exception(self):
+        last_payment_closure_time_timestamp = get_current_utc_timestamp() - 3600 * 24 * 10  # 10 days ago
+        with mock.patch(
+            'core.payments.validation.get_list_of_payments',
+            return_value=_prepare_forced_payment_event_list_with_younger_payments(
+                last_payment_closure_time_timestamp)):
+            with self.assertRaises(PaymentTimestampError):
+                validate_that_last_closure_time_is_older_than_oldest_payment(
+                    'any_key', 'any_key', last_payment_closure_time_timestamp
```

Comment:
Same here. It's hard to see where the `patch()` ends and where the code inside the `with` starts. According to our style it should be:

``` python
        with mock.patch(
            'core.payments.validation.get_list_of_payments',
            return_value=_prepare_forced_payment_event_list_with_younger_payments(
                last_payment_closure_time_timestamp
            )
        ):
            with self.assertRaises(PaymentTimestampError):
                validate_that_last_closure_time_is_older_than_oldest_payment(
                    'any_key',
                    'any_key',
                    last_payment_closure_time_timestamp
                )
```

Even though the difference is pretty minimal, you can immediately see where one nesting level ends and another begins.